### PR TITLE
Added Test Projects phase 1

### PR DESCRIPTION
**Pull request by ShreyasRmsft**
Added tests for full dot net db and w/o db
Added tests for dotnet core db and w/o db

### CODE REVIEW

File: dotnet/aspnet/mssqldb/Application/SampleWebApplication.FunctionalTests/SampleWebApplication.FunctionalTests.csproj [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  <Import Project="..\packages\MSTest.TestAdapter.1.3.2\build\net45\MSTest.TestAdapter.props" Condition="Exists('..\packages\MSTest.TestAdapter.1.3.2\build\net45\MSTest.TestAdapter.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{7D15A631-042A-4C9D-BA57-FF60AA7F48C8}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>SampleWebApplication.FunctionalTests</RootNamespace>
+    <AssemblyName>SampleWebApplication.FunctionalTests</AssemblyName>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```
Comment (sachinma) [inline dotnet/aspnet/mssqldb/Application/SampleWebApplication.FunctionalTests/SampleWebApplication.FunctionalTests.csproj]: Is it same as the target framework version of the web app?

Comment (sachinma) [reply dotnet/aspnet/mssqldb/Application/SampleWebApplication.FunctionalTests/SampleWebApplication.FunctionalTests.csproj]: check other places too

Comment (ShreyasRmsft) [reply dotnet/aspnet/mssqldb/Application/SampleWebApplication.FunctionalTests/SampleWebApplication.FunctionalTests.csproj]: Pasted the wrong folder while creating a new branch. Fixing this in the next commit in a few mins

Comment (ShreyasRmsft) [reply dotnet/aspnet/mssqldb/Application/SampleWebApplication.FunctionalTests/SampleWebApplication.FunctionalTests.csproj]: Fixed

**Comment by ShreyasRmsft** [comment]
Yes, needed 2.1 for selenium compat

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Pull request merged**